.pylintrc

```diff
@@ -1,6 +1,8 @@
 [MASTER]
 init-hook='import sys; sys.path.append("tests/pylint")'
 load-plugins=py2_compat
+# Min Python version to use for version dependend checks.
+py-version=2.7
```

Comment:
Is it correct to set Python 2.7 here? May be 2.6 or 3.5?

Reply:
2.6 would be better for now, yeah.